docs/node/run-your-node/advanced/remote-signer.mdx

```diff
@@ -0,0 +1,258 @@
+
+### Initialize Remote Signer
+
+On your remote signer system, create a directory for the remote signer, e.g.
+`remote-signer`, by running:
+
+```
+mkdir --mode=700 remote-signer
+```
+
+Then initialize the remote signer's keys (i.e. `beacon.pem`, `consensus.pem`,
+`consensus_pub.pem`, `identity.pem`, `identity_pub.pem`, ...) and the server
+certificate by running:
+
+```
+oasis-remote-signer init --datadir remote-signer/
+```
+
+Also, initialize the remote signer's client certificate which will be used by
+the Oasis node to connect to the remote signer:
+
+```
+oasis-remote-signer init_client --datadir remote-signer/
+```
+
+### Run Remote Signer
+
+Choose the gRPC port on which the remote signer will listen for client requests
+and run:
+
+```
+GRPC_PORT=<REMOTE-SIGNER-PORT>
+oasis-remote-signer \
+--datadir remote-signer \
+--client.certificate remote-signer/remote_signer_client_cert.pem \
+--grpc.port $GRPC_PORT \
+--log.level DEBUG
+```
+
+:::tip
+
+The Oasis Remote Signer is configured to run in the foreground by default.
+
+We recommend you configure and use it with a process manager like [systemd] or
+[Supervisor].
+
+:::
+
+[systemd]: https://github.com/systemd/systemd
+[Supervisor]: http://supervisord.org
+
+### Copy Remote Signer Certificate and Client Key and Certificate
+
+For the Oasis node system to be able to connect to the Oasis Remote Signer on
+the remote signer system and to be able to demonstrate its authenticity to it,
+you need to copy the following files from your remote signer system to your
+Oasis node system:
+
+* `remote-signer/remote_signer_server_cert.pem`: The remote signer's
+  certificate. This certificate ensures the Oasis node system is connecting to
+  the trusted remote signer system.
+* `remote-signer/remote_signer_client_key.pem`: The remote signer's client key.
+  This key enables the Oasis node system to demonstrate its authenticity when it
+  is requesting signatures from the remote signer system.
+* `remote-signer/remote_signer_client_cert.pem`: The remote signer's client
+  certificate. This certificate is the counterpart of the remote signer's client
+  key.
+
+## Set Up Oasis Node
+
+For setting up your Oasis node, follow the [Run a Validator
+Node][validator-node] chapter.
+
+:::info
+
+Make sure you've copied the remote signer's certificate and remote signer's
+client key and certificate to your Oasis Node system as described in
+[Copy Remote Signer Certificate and Client Key and Certificate](#copy-remote-signer-certificate-and-client-key-and-certificate) section.
+
+:::
+
+### Initialize Oasis Node
+
+When [initializing your Oasis node](../validator-node.mdx#initializing-a-node),
+you need to pass appropriate `--signer.*` flags to configure the composite and
+remote signers. For example:
+
+```
+ENTITY_DIR=<YOUR-ENTITY-DIR>
+STATIC_IP=<YOUR-STATIC-IP>
+REMOTE_SIGNER_IP=<YOUR-REMOTE-SIGNER-IP>
+REMOTE_SIGNER_PORT=<YOUR-REMOTE-SIGNER-PORT>
+oasis-node registry node init \
```

Comment:
This is not needed anymore and should be removed, similarly to how we removed this in the validator node docs.